assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -27,24 +25,18 @@ const useInitQueries = ({ eventId }) => {
 		},
 	});
 
-	if (!initialized) {
-		toaster.loading(loadingDates, toasterDatesMessage);
-		toaster.error(datetimeError);
-	}
-
 	const datetimes = pathOr([], ['datetimes', 'nodes'], datetimesData);
 	const datetimeIn = datetimes ? datetimes.map(({ id }) => id) : [];
-
 	const toasterTicketsMessage = 'initializing event editor tickets';
-	const [
-		fetchRelatedTickets,
-		{ data: ticketsData, error: ticketError, loading: loadingTickets, called },
-	] = useLazyQuery(GET_TICKETS, {
+	const skipTicketsQuery = datetimeIn.length === 0;
+
+	const { data: ticketsData, error: ticketError, loading: loadingTickets } = useQuery(GET_TICKETS, {
 		variables: {
 			first: 50,
 			where: {
 				datetimeIn,
 			},
+			skip: skipTicketsQuery,
```

Comment:
On the other hand, passing `skip` true gives me this result:
![image](https://user-images.githubusercontent.com/1477453/70322185-118b6180-1832-11ea-9654-1c0d276f13d0.png)
@manzoorwanijk is something that needs to be done on serverside ?

Reply:
Also `skip` option is a nice declarative way of doing this chaining of queries.